examples/lib/language_tour/classes/employee.dart

```diff
@@ -0,0 +1,47 @@
+// ignore_for_file: unnecessary_cast
+// #docplaster
+import '../util/print.dart';
+
+Map getDefaultData() => {}; // stub
+
+// #docregion
+class Person {
+  String firstName;
+
+  Person.fromJson(Map data) {
+    $print('in Person');
```

Comment:
All code excerpts use `$print()` instead of `print()`. The global `$print` variable (defined in `../util/print.dart`) is overwritten when running tests so that we can capture print output.

Of course, when displaying code excerpts in the Language Tour, we show `print()` instead. The substitution is done by `scripts/refresh-code-excerpts.sh`.

Reply:
Done